.github/workflows/nightly-build.yaml

```diff
@@ -2,7 +2,7 @@
   name: "Nightly build",
   on: {
     schedule: [
-      { cron: "20 17 * * *" }
+      { cron: "05 7 * * *" }
```

Comment:
Does the `5` need to be prefixed with `0`? Also, in which timezone does this run?

Reply:
I suppose I should either do both or neither, huh?  😅 

Also, UTC!